test/test_memmove.c

```diff
@@ -0,0 +1,203 @@
+#include <stdio.h> 
+#include <string.h> 
+#include "test_common.h"
+
+#define SEED 1581072278
+#define BUF_LEN 4000
+#define BYTES_COPIED 256
+#define OFFSET_VARIANCE 4
+#define SIZE_VARIANCE 4
+#define BUF_DUMP_WIDTH 20
+
+static const int test_offsets[][2] = {
```

Comment:
You use this values only once each in code. I think explicit offsets will be cleaner.